gatsby-config.js

```diff
@@ -95,6 +97,68 @@ module.exports = {
     'gatsby-transformer-json',
     'gatsby-transformer-sharp',
     'gatsby-plugin-sharp',
+    {
+      resolve: `gatsby-plugin-feed`,
+      options: {
+        query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
+              }
+            }
+          }
+        `,
+        feeds: [
+          {
+            serialize: ({ query: { site, allMarkdownRemark } }) => {
```

Comment:
Output llooks great (tested with Feedbro on Firefox)
![Screen Shot 2021-08-31 at 15 50 13](https://user-images.githubusercontent.com/1701755/131514547-e804c612-e532-428b-81c3-74b4912510ce.png)

Funny thing. The images are not loaded of course so it's displaying the fallback blurred image. I'm not sure if we can give the rss some better image easily without affecting our loading mechanism.
![Screen Shot 2021-08-31 at 15 50 42](https://user-images.githubusercontent.com/1701755/131514612-aec7d9fb-8b19-4cd5-8ef7-c77141c33a9b.png)